qcodes/instrument/delegate/delegate_instrument.py

```diff
@@ -95,24 +99,32 @@ def __init__(
         self,
         name: str,
         station: Station,
-        parameters: Union[Mapping[str, Sequence[str]], Mapping[str, str]],
+        parameters: Optional[Union[Mapping[str, Sequence[str]], Mapping[str, str]]] = None,
+        channels: Optional[Union[Mapping[str, Mapping[str, Any]], Mapping[str, str]]] = None,
```

Comment:
One concern is that if anyone passes arguments after this one as positional arguments that will break with this change. 
We could consider making all these arguments kwargs only by adding a `*` (probably after station)
@astafan8 what do you think

Reply:
either way it would be a breaking change (with varying degree of breakage), so including the ``*`` will make it more evident which is generally nice. however, there are not so many users of this feature (yet) hence we can probably skip adding the ``*`` at this point. so i'd vote for not adding the star but do note the "breaking change"